### PR TITLE
[SAR-10466] Update to accept new en language block

### PR DESCRIPTION
**Pull request by refaque-ahmad**

### CODE REVIEW

File: it/uk/gov/hmrc/minorentityidentificationfrontend/views/overseasViews/CaptureUtrViewTests.scala [hunk 1]
```diff
@@ -131,4 +131,13 @@ trait CaptureUtrViewTests {
...
 
+  def testServiceName(serviceName: String, result: => WSResponse): Unit = {
+    lazy val doc: Document = Jsoup.parse(result.body)
+
+    "correctly display the service name" in {
+      doc.getServiceName.text mustBe serviceName
+    }
+  }
+
+
```
Comment (slavvco) [inline it/uk/gov/hmrc/minorentityidentificationfrontend/views/overseasViews/CaptureUtrViewTests.scala:142]: unnecessary second empty line

File: it/uk/gov/hmrc/minorentityidentificationfrontend/views/trustViews/TrustCaptureUtrViewTests.scala [hunk 2]
```diff
@@ -131,4 +131,13 @@ trait TrustCaptureUtrViewTests {
...
 
+  def testServiceName(serviceName: String, result: => WSResponse): Unit = {
+    lazy val doc: Document = Jsoup.parse(result.body)
+
+    "correctly display the service name" in {
+      doc.getServiceName.text mustBe serviceName
+    }
+  }
+
+
```
Comment (slavvco) [inline it/uk/gov/hmrc/minorentityidentificationfrontend/views/trustViews/TrustCaptureUtrViewTests.scala:142]: unnecessary second empty line

File: it/uk/gov/hmrc/minorentityidentificationfrontend/views/uaViews/CaptureCHRNumberViewTests.scala [hunk 3]
```diff
@@ -176,4 +176,13 @@ trait CaptureCHRNumberViewTests {
...
 
+  def testServiceName(serviceName: String, result: => WSResponse): Unit = {
+    lazy val doc: Document = Jsoup.parse(result.body)
+
+    "correctly display the service name" in {
+      doc.getServiceName.text mustBe serviceName
+    }
+  }
+
+
```
Comment (slavvco) [inline it/uk/gov/hmrc/minorentityidentificationfrontend/views/uaViews/CaptureCHRNumberViewTests.scala:187]: unnecessary second empty line

File: test/uk/gov/hmrc/minorentityidentificationfrontend/utils/MessageHelperSpec.scala [hunk 4]
```diff
@@ -35,7 +35,7 @@ class MessageHelperSpec extends AnyFlatSpec with BeforeAndAfterEach {
     val welshServiceName = "This is a welsh service name"
     val englishServiceName = "This is an english service name"
 
-    val welshLabels: JourneyLabels = JourneyLabels(welshServiceName)
+    val welshLabels: JourneyLabels = JourneyLabels(Some(welshServiceName), Some(englishServiceName))
```
Comment (slavvco) [inline test/uk/gov/hmrc/minorentityidentificationfrontend/utils/MessageHelperSpec.scala:38]: journeyLabels would be better name, as labels now can be both welsh and english

**Review by slavvco — CHANGES_REQUESTED** [review]
Looks good, but I left few comments.

File: it/uk/gov/hmrc/minorentityidentificationfrontend/controllers/ControllersSupportWelshTranslationISpec.scala [hunk 5]
```diff
@@ -97,7 +97,7 @@ class ControllersSupportWelshTranslationISpec
             extractDocumentFrom(aWSResponse = get(getUrlToBeTested, cookie = cyLangCookie))
           }
 
-          actualDocFromResponse.getServiceName.text mustBe testDefaultWelshJourneyConfig.pageConfig.optLabels.get.welshServiceName
+          actualDocFromResponse.getServiceName.text mustBe testDefaultWelshJourneyConfig.pageConfig.optLabels.get.optWelshServiceName.getOrElse(None)
```
[the remark this answers is not in the document]
Comment (refaque-ahmad) [reply it/uk/gov/hmrc/minorentityidentificationfrontend/controllers/ControllersSupportWelshTranslationISpec.scala]: done

File: it/uk/gov/hmrc/minorentityidentificationfrontend/views/overseasViews/CaptureUtrViewTests.scala (hunk 1 above)
Comment (refaque-ahmad) [reply it/uk/gov/hmrc/minorentityidentificationfrontend/views/overseasViews/CaptureUtrViewTests.scala]: done

File: it/uk/gov/hmrc/minorentityidentificationfrontend/views/trustViews/TrustCaptureUtrViewTests.scala (hunk 2 above)
Comment (refaque-ahmad) [reply it/uk/gov/hmrc/minorentityidentificationfrontend/views/trustViews/TrustCaptureUtrViewTests.scala]: done

File: it/uk/gov/hmrc/minorentityidentificationfrontend/views/uaViews/CaptureCHRNumberViewTests.scala (hunk 3 above)
Comment (refaque-ahmad) [reply it/uk/gov/hmrc/minorentityidentificationfrontend/views/uaViews/CaptureCHRNumberViewTests.scala]: done

File: test/uk/gov/hmrc/minorentityidentificationfrontend/utils/MessageHelperSpec.scala (hunk 4 above)
Comment (refaque-ahmad) [reply test/uk/gov/hmrc/minorentityidentificationfrontend/utils/MessageHelperSpec.scala]: done

**Review by slavvco — APPROVED** [review]
All looks ok to me.

**Pull request merged**